src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -605,6 +621,211 @@ void copy_columns_by_row_chunks(const ov::Tensor& src, ov::Tensor& dst) {
 namespace ov {
 namespace genai {
 
+SMStaticLLMPipeline::SMStaticLLMPipeline(
+    const std::filesystem::path& models_path,
+    const ov::genai::Tokenizer& tokenizer,
+    const std::string& device,
+    const ov::AnyMap& config
+) : LLMPipelineImplBase(tokenizer,
+                        utils::from_config_json_if_exists(models_path)) {
+    ov::Core core;
+    auto model = core.read_model((models_path / "openvino_model.xml").string());
+
+    ov::AnyMap properties = config;
+
+    const uint32_t kMaxPromptLen = pop_int_and_cast(properties, "MAX_PROMPT_LEN").value_or(1024u);
+    const uint32_t kMinResponseLen = pop_int_and_cast(properties, "MIN_RESPONSE_LEN").value_or(128u);
```

Comment:
see the above code, it was aligned to 64. Probably it makes sense to unify how these options are handled between the two classes (without overdesign)

Reply:
Yes, I did the alignment inside of the `LLMCompiledModel`  constructor in the NPUW. Do you think I need to remove it there and instead do alignment here? I did it in `LLMCompiledModel`, since I thought it might be of implementation detail..

Reply:
What do you think?